man/virt-xml.rst

```diff
@@ -152,6 +152,16 @@ These options decide what action to take after altering the XML. In the common c
 
     By default this also implies ``--define``.
 
+``--update-force``
+    **EXPERIMENTAL:** Forces a device update using libvirt's ``VIR_DOMAIN_DEVICE_MODIFY_FORCE`` flag.
```

Comment:
I'm not sure about mentioning "experimental": why would it be so? is the implementation of this CLI option in virt-install experimental, or the underlying behaviour of it?